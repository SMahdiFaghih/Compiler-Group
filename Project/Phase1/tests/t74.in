aint apublic public a publica privateb private void avoid