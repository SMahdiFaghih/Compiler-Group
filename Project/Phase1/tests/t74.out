T_ID aint
T_ID apublic
public
T_ID a
T_ID publica
T_ID privateb
private
void
T_ID avoid